### ISSUE TITLE
[BUG] Selected Form Sheet: Tab key navigation doesn't work as expected

### ISSUE THREAD

**Issue by PeterMarinov** [endatix/endatix#215]
**Describe the bug**
The tab key on the Form Details Sheet is working in a very strange way and is not usable for visually impaired people or just keyboard navigation

**To Reproduce**

1. Go to Forms List page
2. Click on form with Submission to open the Form Details Sheet
3. Start pressing the Tab key on the keyboard
4. Observe the focus zone behavior and note the following issues:
   - First focused element is the second button - the Preview button
   - It takes 2 key presses to move to the next element
   - After the Preview Button, the focus moves to the Delete button, then to the close button and then it starts from the Edit button again, which is wrong

**Expected behavior**
Tab navigation starts from the first element in the DOM and continues in linear fashion. Also it should take one Tab press to move to the next button.

**Screenshots**
https://github.com/user-attachments/assets/01f7accc-6da9-4f91-8f65-752932c16680

**Desktop (please complete the following information):**
 - N/A
